### ISSUE TITLE
Ldap integration?

### ISSUE THREAD

**Issue by wimvandenborre** [intuitem/ciso-assistant-community#472]
Hey, would be awesome to have ldap integration :)

**Comment by eric-intuitem** [comment]
Hi @wimvandenborre, can you elaborate on the expected result? Would you like using LDAP for identification and authentication, and/or for autorisation (group membership management)?

**Comment by wimvandenborre** [comment]
Correct, would be nice to be able to login with ldap accounts and also for group membership management

**Comment by ab-smith** [comment]
Related to #366

**Comment by ab-smith** [comment]
hello @wimvandenborre  regarding our design of SSO, we are going for now with Authentication on external IdP (eg. Entra) and Authorisation on the application.
Later this year, we will consider mapping the roles to AAD groups.

When you mentioned LDAP, were you thinking about AAD/Entra or another implementation? The current setup supports SAML, by the way, so whatever compatible solution should be fine.

**Comment by aremydata** [comment]
Hello, I have a quick question to ask. I've linked CISO to our LDAP (ENTRA), and the connection works well, but once it's initiated, it redirects me to CISO, where I encounter a 500 error page. I can't figure out where it's coming from. For your information, I'm using CISO 1.9.12 and using NGINX instead of Caddy. Thanks for your help; if anyone has encountered this problem before, I'm available to provide more information if needed.

**Comment by ab-smith** [comment]
Hello,
For LDAP, there are no plan to support directly for the SSO. We've included OIDC support recently which would cover most IdP.
For the ENTRA settings, I believe this was discussed and fixed on Discord. A Detailed step-by-step guide is available on the documentation.
Regards,
